packages/nimble-components/src/wafer-map/tests/wafer-map.stories.ts

```diff
@@ -239,7 +280,46 @@ const metadata: Meta<WaferMapArgs> = {
                     set4: 'Many dies are highlighted'
                 }
             },
-            defaultValue: 'set1'
+            defaultValue: 'set1',
+            if: { arg: 'apiVersion', eq: 'stable' }
```

Comment:
The ability to hide parameters based on the value of another one is neat. But I think it's a bit subtle. I'd suggest:
1. adding a note to both `dies` and `diesTable` saying which API they're part of
2. moving the apiVersion parameter to be first in the list so it's obvious it controls everything about the component
3. giving the apiVersion parameter a display name like "API Version" so it's obvious it isn't a property of the component
4. updating the description of apiVersion to mention what controls it in code. Something like: "the component will use the stable API if data is provided via `dies` and the experimental API if data is provided via `diesTable`"